_posts/developers/pod-server/2019-01-01-00_overview.md

```diff
@@ -22,104 +22,8 @@ The following **open-source servers** are available:
 * [solid-nextcloud](https://github.com/pdsinterop/solid-nextcloud) as a plugin for Nextcloud
 * [PHP Solid Server](https://github.com/pdsinterop/php-solid-server) for PHP
 
-We walk you through the installation of the Community Solid Server below.
-<br>
-We also provide [Instructions for Node Solid Server](/self-hosting/nss).
-
-# Running the Community Solid Server {#running-css}
-To run the Community Solid Server,
-you can choose between:
-
-- [Node.js](#nodejs)
-- [Docker](#docker)
-
-## Via Node.js and npm {#nodejs}
-Ensure you have [installed Node.js](https://nodejs.org/en/download/).
-We support versions 12.7 and up,
-but recommend 14 or 16.
-<br>
-To **install the server**, open a command prompt and type:
-
-```shell
-npm install -g @solid/community-server
-```
-
-To **run the server** with in-memory storage, type:
-```shell
-community-solid-server
-```
-
-To run the server with **file-based storage**, type:
-```shell
-community-solid-server -c @css:config/file.json
-```
-
-You can now [access your server](#access) locally.
-
-## Via Docker {#docker}
-[Docker](https://docs.docker.com/get-docker/) allows you to run the server
-without having Node.js installed:
-```shell
-# Clone the repo to get access to the configs
-git clone https://github.com/CommunitySolidServer/CommunitySolidServer.git
-cd CommunitySolidServer
-# Run the image, serving your `~/Solid` directory on `http://localhost:3000`
-docker run --rm -v ~/Solid:/data -p 3000:3000 -it solidproject/community-server:latest
-# Or use one of the built-in configurations
-docker run --rm -p 3000:3000 -it solidproject/community-server -c config/default.json
-# Or use your own configuration mapped to the right directory
-docker run --rm -v ~/solid-config:/config -p 3000:3000 -it solidproject/community-server -c /config/my-config.json
-```
-
-## Accessing your server {#access}
-Open your browser and visit [http://localhost:3000/](http://localhost:3000/).
-You will be greeted by a welcome page.
-
-This welcome page is hosted on a _root Pod_,
-to which **you and your apps can read and write**.
+See [the features matrix](https://github.com/solid/test-suite#table) to find out which server implementation supports which required and/or optional parts of the Solid spec.
```

Comment:
This is new information that I think is outside the scope of the issue that this PR intends to fix/resolve. I suggest to omit this for the time being.

```suggestion
```